### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by madmaxio** [komora-io/marble#3]
Hi, would you recommend use this library somewhere or it is some kind of experimental/research project?
Can it be considered sled replacement (for some use cases) and what are your thoughts about sled's future?

**Comment by spacejam** [comment]
Marble is sled's future storage engine for things that get cooled from the log (https://github.com/komora-io/sharded-log is the log that sled will use).

I've tried to keep Marble simple enough, while still subjecting it to fuzzing and interesting concurrent crash testing. I'm about to cut a new release that I'm comfortable telling people to try out in interesting use cases :)

**Comment by madmaxio** [comment]
Thanks for reply, will be very interesting for sled users I think.

**Comment by RockyGitHub** [comment]
Yes!  I am using Sled and would really like to try out what future it has

**Comment by tv42** [comment]
Just because I think many haven't noticed this little detail: Sled is MIT & Apache2, but everything in Komora seems to be GPL3. This... definitely limits my future use of Sled :-(

**Comment by tv42** [comment]
Update: it seems this is switching from GPL3 to Apache-2: https://github.com/komora-io/marble/commit/c599ecb2e75a26a37660ca28a0a02e36ce8dbfab

Looks like most other komora-io things are Apache-2 now too, except https://github.com/komora-io/sharded-log is still GPL-3.
